src/lib/BasicHead.tsx

```diff
@@ -62,152 +49,119 @@ function BasicHead(props: any) {
   const { seneca } = ctx()
 
   const {
-    frame,
+    frame
   } = spec
 
-  // console.log('BasicHead.spec', spec)
-
   // spec schema definition with Gubu
-  const shape = Gubu({
+  const shape = Gubu(Open({
```

Comment:
let's call this `BasicHeadSpecShape` and place outside the function, as we only need to create it once

Reply:
and same for the other components